app/components/primer/timeline_item_component.html.erb

```diff
@@ -1,4 +1,10 @@
 <%= render Primer::BaseComponent.new(**kwargs) do %>
+  <% if avatar %>
+    <%= render Primer::BaseComponent.new(**avatar.kwargs) do %>
```

Comment:
Do we need this wrapping component, or could we pass the avatar `kwargs` directly to `AvatarComponent`?